tiledb/sm/serialization/array_schema.cc

```diff
@@ -210,7 +210,10 @@ Status attribute_from_capnp(
 
   // Set the fill value.
   auto fill_value = attribute_reader.getFillValue();
-  (*attribute)->set_fill_value(fill_value.asBytes().begin(), fill_value.size());
+  if (fill_value.size() > 0) {
```

Comment:
Actually you might also consider the `attribute_reader.hasFillValue()`

Reply:
That's much better, will do. Thanks.